src/components/DeleteEditButtons/index.js

```diff
@@ -2,103 +2,75 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 
-import {
-  Button,
-  Dialog,
-  DialogActions,
-  DialogContent,
-  DialogContentText,
-  DialogTitle,
-} from '@material-ui/core';
+import { Button } from '@material-ui/core';
 import { withStyles } from '@material-ui/core/styles';
 import DeleteIcon from '@material-ui/icons/Delete';
 import EditIcon from '@material-ui/icons/Edit';
 import styles from './styles';
 
 import * as ROUTES from '../../constants/routes';
 import { deleteEventById } from '../../services/events';
+import { ConfirmModal } from "../modals";
 
-const INITIAL_STATES = {
-  isConfirmationModalOpen: false,
-};
-
-class DeleteEditButtons extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { ...INITIAL_STATES };
-  }
-
-  handleModalOpen = () => {
-    this.setState({ isConfirmationModalOpen: true });
-  };
-
-  handleModalClose = () => {
-    this.setState({ isConfirmationModalOpen: false });
-  };
+const DeleteEditButtons = props => {
+  const { classes, eventId } = props;
 
-  handleDeleteEvent = eventId => {
-    deleteEventById(eventId)
+  const handleDeleteEvent = eventToDeleteId => {
+    deleteEventById(eventToDeleteId)
       .then(res => {
         return res;
       })
       .catch(err => {
         console.log(err);
       });
+  };
 
-    this.handleModalClose();
+  const openButtonProps = {
+    openButtonName: 'Delete',
+    openButtonOnClick: null,
+    className: classes.delete,
+    startIcon: <DeleteIcon />,
+    primary: true,
+    negative: true,
   };
 
-  render() {
-    const { classes, eventId } = this.props;
-    const { isConfirmationModalOpen } = this.state;
+  const confirmButtonProps = {
+    buttonName: 'Yes',
+    onClick: () => handleDeleteEvent(eventId),
```

Comment:
can we overload the onClick with this.history.push or something so we don't need the component and to?